modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/XMLDTDVersionCheck.java

```diff
@@ -84,6 +58,28 @@ private String _checkDTDVersion(String content) throws IOException {
 			matcher.start());
 	}
 
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws IOException {
+
+		_readReleaseProperties();
+
+		if ((_releaseProperties == null) && !fileName.endsWith(".xml")) {
```

Comment:
It should be `(_releaseProperties == null) || !fileName.endsWith(".xml")`. This is why we are seeing this CI test failure:

```log
     [java] 1: ./modules/apps/translation/translation-service/src/main/resources/com/liferay/translation/internal/security/permission/dependencies/resource-actions.xml.tpl expected:<...ce Action Mapping 7.[4.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_4]_0.dtd">
     [java] 
     [java] <resource-...> but was:<...ce Action Mapping 7.[3.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_3]_0.dtd">
```

Reply:
Thanks! Already fixed. Running CI again.